.github/workflows/canary-integration-test.yml

```diff
@@ -609,6 +615,7 @@ jobs:
         uses: ./.github/workflows/tmate_debug
         with:
           use-tmate: ${{ secrets.USE_TMATE }}
+          limit-access-to-actor: true
```

Comment:
You can remove this we already get this in in a seprate PR

Reply:
Yes will get rid of it, I added so I can debug if any issue in osd on tmate session.